hbase/pom.xml

```diff
@@ -61,5 +61,13 @@
       </plugin>
     </plugins>
   </build>
+  <profiles>
+    <profile>
+      <id>hbase-2.x</id>
```

Comment:
HBase 1.x is EOL, see HBASE-27286, I think we don't need a profile, just adapt to HBase 2.x

Reply:
Looks good!

Reply:
Hi @pan3793 ,

Great. I will simply upgrade it.